public/locales/en-US/common.json

```diff
@@ -0,0 +1,55 @@
+{
+  "step": "{{count}} Steps",
+  "hike_version": "Hike Version: ",
+  "platform_version": "Platform Version",
+  "categories": "Categories",
+  "start": "START",
+  "mobile_app_title": "Build Your First Mobile App",
+  "mobile_app_desc": "<p>Create a mobile app incredibly fast, without coding. Learn how to drag and drop UI (user interface) elements into an app, integrate your app with backend data, and secure your app with an authentication mechanism.</p>",
+  "mobile_app_1_title": "Launch your First Mobile App",
+  "mobile_app_1_desc": "Open a native mobile app and preview it on your mobile device.",
+  "mobile_app_1_detail": "<p><!--StartFragment--></p> <p><br /> This learning module is a Volt MX HIKE. &nbsp;A HIKE is an interactive tour inside Volt MX Iris that guides you through step-by-step process of each module. You can access the HIKES catalog by clicking the Hike icon in the Iris toolbar:&nbsp;</p> <p> &nbsp;</p> <p>Learn how to:&nbsp;</p> <p>&nbsp; &nbsp; Import a sample native app that shows lists of Accounts and Recipients&nbsp;</p> <p>&nbsp; &nbsp; Preview the sample app on a mobile device&nbsp;</p> <p><!--EndFragment--></p>",
+  "mobile_app_1_time": "11 Mins",
+  "mobile_app_1_file_url": "https://community.qa-kony.com/hikes/tour/22913/download/15236",
+  "mobile_app_1_file_name": "launchyourfirstmobileapp.zip",
```

Comment:
would it make sense (or be possible) to make the string resources follow the same object array pattern as the tours.json or even a referential array so that we don't have them statically sequential?   if a tour changes position in the definition array, the number won't make sense....

example: 
```suggestion
"tourRes": {
  "24825": {
    "title": "Launch your First Mobile App",
    "desc": "Open a native mobile app and preview it on your mobile device.",
    "detail": "<p><!--StartFragment--></p> <p><br /> This learning module is a Volt MX HIKE. &nbsp;A HIKE is an interactive tour inside Volt MX Iris that guides you through step-by-step process of each module. You can access the HIKES catalog by clicking the Hike icon in the Iris toolbar:&nbsp;</p> <p> &nbsp;</p> <p>Learn how to:&nbsp;</p> <p>&nbsp; &nbsp; Import a sample native app that shows lists of Accounts and Recipients&nbsp;</p> <p>&nbsp; &nbsp; Preview the sample app on a mobile device&nbsp;</p> <p><!--EndFragment--></p>",
    "time": "11 Mins",
    "file_url": "https://community.qa-kony.com/hikes/tour/22913/download/15236",
    "file_name": "launchyourfirstmobileapp.zip"
    },
  "nid of app 2...": {
    // mobile app 2 resources...
  },
```